content/en/monitors/faq/anomaly-monitor.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Anomaly Monitor FAQ
+kind: faq
+---
+
+### Should I use anomaly detection for everything?
+
+No. Anomaly detection is designed to assist with visualizing and monitoring metrics that have predictable patterns. For example, `my_site.page_views{*}` might be driven by user traffic and thus vary predictably by time of day and day of week. If your metric does not have a repeated or predictable pattern, then a simple chart overlay or threshold alert might be better.
```

Comment:
```suggestion
No. Anomaly detection is designed to assist with visualizing and monitoring metrics that have predictable patterns. For example, you wouldn't want to use Anomaly detection if `my_site.page_views{*}` is driven by user traffic that varies predictably by time of day and day of week. If your metric has a repeated or predictable pattern, then a chart overlay or threshold alert might be better.
```

this last part seems like it should be if it does have a predictable pattern, but I am not sure. This example is oddly phrased, so I am just trying to make it a bit more clear with my suggestion.

Reply:
Agreed. `my_site.page_views` is a metric that works well with anomaly detection. I will reword this.